spec/scrabble-spec.js

```diff
@@ -1,54 +1,114 @@
 // Note: you shouldn't need to change anything in this file.
 
-Scrabble = require('../src/scrabble')
+Scrabble = require("../src/scrabble");
 let scrabble;
 
-describe("Scrabble", function() {
-  it('returns 0 for empty words', function() {
-    scrabble = new Scrabble('')
-
-    expect(scrabble.score()).toEqual(0)
-  })
-
-  it('returns 0 for whitespace', function() {
-    scrabble = new Scrabble(" \t\n")
-
-    expect(scrabble.score()).toEqual(0)
-  })
-
-  it('returns 0 for null', function() {
-    scrabble = new Scrabble(null)
-
-    expect(scrabble.score()).toEqual(0)
-  })
-
-  it('scores short word', function() {
-    scrabble = new Scrabble('a')
-
-    expect(scrabble.score()).toEqual(1)
-  })
-
-  it('scores short word', function() {
-    scrabble = new Scrabble('f')
-
-    expect(scrabble.score()).toEqual(4)
-  })
-
-  it('scores a simple word', function() {
-    scrabble = new Scrabble('street')
-
-    expect(scrabble.score()).toEqual(6)
-  })
-
-  it('scores a more complicated word', function() {
-    scrabble = new Scrabble('quirky')
-
-    expect(scrabble.score()).toEqual(22)
-  })
-
-  it('scores a case-insensitive word', function() {
-    scrabble = new Scrabble('OXYPHENBUTAZONE')
-
-    expect(scrabble.score()).toEqual(41)
-  })
-})
+describe("Scrabble", function () {
+  it("returns 0 for empty words", function () {
+    scrabble = new Scrabble("");
+
+    expect(scrabble.score()).toEqual(0);
+  });
+
+  it("returns 0 for whitespace", function () {
+    scrabble = new Scrabble(" \t\n");
+
+    expect(scrabble.score()).toEqual(0);
+  });
+
+  it("returns 0 for null", function () {
+    scrabble = new Scrabble(null);
+
+    expect(scrabble.score()).toEqual(0);
+  });
+
+  it("scores short word", function () {
+    scrabble = new Scrabble("a");
+
+    expect(scrabble.score()).toEqual(1);
+  });
+
+  it("scores short word", function () {
+    scrabble = new Scrabble("f");
+
+    expect(scrabble.score()).toEqual(4);
+  });
+
+  it("scores a simple word", function () {
+    scrabble = new Scrabble("street");
+
+    expect(scrabble.score()).toEqual(6);
+  });
+
+  it("scores a more complicated word", function () {
+    scrabble = new Scrabble("quirky");
+
+    expect(scrabble.score()).toEqual(22);
+  });
+
+  it("scores a case-insensitive word", function () {
+    scrabble = new Scrabble("OXYPHENBUTAZONE");
+
+    expect(scrabble.score()).toEqual(41);
+  });
+
+  //Extension criteria Tests
+  //double letter
```

Comment:
These comments are a little moot. You should ensure your describe/it are descriptive, as that is what will be displayed when the test fails.